mlir/lib/Target/LLVMIR/Dialect/OpenMP/OpenMPToLLVMIRTranslation.cpp

```diff
@@ -255,7 +255,6 @@ static LogicalResult checkImplementationStatus(Operation &op) {
       .Case([&](omp::TaskOp op) {
         checkAllocate(op, result);
         checkInReduction(op, result);
-        checkMergeable(op, result);
```

Comment:
nit: this is the only use of `checkMergeable` so it can now be removed

Reply:
Just double confirming: do you want me to add a comment here? I removed this check, because mergeable is a `task` specific clause only.

Reply:
I meant that you should also remove the definition of `checkMergable` as the lambda is now unused.